master-firmware/src/scara/scara.c

```diff
@@ -8,6 +8,18 @@
 #include "scara_port.h"
 #include "scara_jacobian.h"
 
+static void scara_shutdown_joints(scara_t* arm);
+
+static void scara_lock(mutex_t* mutex)
```

Comment:
[question] Any reason to put this in a separate function? It does not improve portability or anything?

Reply:
You're right, It should probably move to `scara_port` to improve portability. I just found it was more readable.